Gemfile

```diff
@@ -13,7 +13,7 @@ gem 'pg'
 # Use Puma as the app server
 gem 'puma', '~> 3.11'
 
-gem 'blacklight', github: 'projectblacklight/blacklight'
+gem 'blacklight', github: 'projectblacklight/blacklight', branch: 'avoid_unnecessary_lookup'
```

Comment:
TODO: Add ticket for undoing this.

Reply:
https://github.com/sul-dlss-labs/rialto-webapp/issues/93